src/main/java/dk/bec/polonez/reservationsystem/service/OfferService.java

```diff
@@ -35,22 +38,48 @@ public Offer getById(long id) {
     }
 
     public ResponseOfferDto addOffer(CreateOfferDto offerDto) {
-        User user = new User();
-        user.setId(1L);
-
-        Offer.OfferBuilder builder = Offer.builder();
-        Offer offer = builder
+        User owner = userRepository.getById(offerDto.getOwnerId());
+        Offer offer = Offer.builder()
                 .name(offerDto.getName())
-                .owner(user)
+                .owner(owner)
```

Comment:
nie wiem jaki macie model bazodanowy, ale troche dziwne, ze trzeba calego user-a dodac do oferty, obstawiam, ze wszystko co offer musi wiedziec o user-e to id

Reply:
chcieliśmy mieć klucze obce jako całe obiekty żeby mieć krótszy dostęp bez używania repozytorium.